air-quality-ui/system_tests/utils/default_aqi_enums.ts

```diff
@@ -33,3 +33,11 @@ export enum CaseAQI6 {
   pm10 = 160,
   so2 = 800,
 }
+
+export enum valueEnum {
```

Comment:
what does this represent?

Reply:
These are the default values I used for city chart measurements. Can change the naming if need be.